test/test_sparse.py

```diff
@@ -1962,6 +1962,53 @@ def _test_basic_ops_hybrid():
         _test_basic_ops()
         _test_basic_ops_hybrid()
 
+    def _test_add_sparse_broadcasting_shape(self, sparse_dim1, nnz1, shape1, sparse_dim2, nnz2, shape2, dtype, device, coalesced):
+        x1, _, _ = self._gen_sparse(sparse_dim1, nnz1, shape1, dtype, device, coalesced)
+        x2, _, _ = self._gen_sparse(sparse_dim2, nnz2, shape2, dtype, device, coalesced)
+        y1 = x1 + x2
+        y2 = x1.clone()
+        y2.add_(x2)
+        expected = self.safeToDense(x1) + self.safeToDense(x2)
+        self.assertEqual(self.safeToDense(y1), expected)
+        self.assertEqual(self.safeToDense(y2), expected)
```

Comment:
Fine with having separate test, but would it not be easier to add these cases to the already existing test suite if possible?